pkg/server/service.go

```diff
@@ -401,6 +401,7 @@ func serveGatewaySecure(ctx context.Context, pluginRegistry *plugins.Registry, c
 			Certificates: []tls.Certificate{*cert},
 			NextProtos:   []string{"h2"},
 		},
+		ReadHeaderTimeout: 0,
```

Comment:
why 0?

Reply:
shouldn't we specify a non-zero value then to address the actual slowloris vulnerability?

Reply:
yes, added it to the server config and used 32s as the default value which is same as the [k8s API server](https://github.com/kubernetes/kubernetes/blob/75b19b242c34f7c1d1545e156f802f0a06b90533/staging/src/k8s.io/apiserver/pkg/server/secure_serving.go#L172).